ci/cron/src/Main.hs

```diff
@@ -463,4 +463,9 @@ main = do
       Docs -> do
           docs sdkDocOpts
           docs damlOnSqlDocOpts
+          -- FIXME: this is meant to run once to create the `latest` file
```

Comment:
the comment sounds wrong, we want to update `snapshots.json`.

Reply:
Also not sure if at this point it isn’t just better to keep this code long term.

Reply:
I'll merge as is to fix the issue tonight; I'll think a bit more about what to do longer-term.

Reply:
:+1: